src/atoms/Button/buttons.module.scss

```diff
@@ -90,7 +91,7 @@ $button-solid-background-tertiary-color:   #f08f5c;
 }
 
 .disabled {
-  composes: button;
+  @extend %button;
```

Comment:
I know we discussed doing it this way, but do you think it might make sense to just actually give all the buttons the `.button` class, instead of extending this?